io.raphamorim.Rio.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "io.raphamorim.Rio",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "rio",
+    "finish-args": [
+        "--filesystem=xdg-config/rio",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
this is no allowed by default

Reply:
tell that to the linter.

https://docs.flathub.org/docs/for-app-authors/linter/

Reply:
but the rational is wrong. A proper terminal is a terminal not a shell runner.